docs/articles/user-guide/cli-commands.md

```diff
@@ -23,31 +23,42 @@ These Salesforce CLI commands are available in Salesforce Extensions for VS Code
 - `force:apex:test:run --resultformat human ...`: **SFDX: Invoke Apex Tests...**
 - `force:apex:trigger:create ...`: **SFDX: Create Apex Trigger**
 - `force:auth:logout --all --noprompt`: **SFDX: Log Out from All Authorized Orgs**
-- `force:auth:web:login --setdefaultdevhubusername`: **SFDX: Authorize a Dev Hub**
+- `force:auth:web:login --setdefaultdevhubusername`, `force:auth:device:login --setdefaultdevhubusername` (in remote development): **SFDX: Authorize a Dev Hub**
 - `force:config:list`: **SFDX: List All Config Variables**
+- `force:config:set --isvDebuggerSid --isvDebuggerUrl --instanceUrl`: **SFDX: Create and Set Up Project for ISV Debugging** (step 2)
+- `force:data:record:create --sobjecttype --values --usetoolingapi`: **SFDX: Turn On Apex Debug Log for Replay Debugger**
+- `force:data:record:delete --sobjecttype --sobjectid --usetoolingapi`: **SFDX: Turn Off Apex Debug Log for Replay Debugger**
```

Comment:
Do we want to include all the flags/args that are being used? Or just the ones that users need to enter values for?

Reply:
Only the relevant ones that a user has to specify for the SFDX command.

Reply:
In that case, I believe we shouldn't include any of the flags for `force:data:record:create` and `force:data:record:delete`

Reply:
@lcampos were you suggesting we should keep the flags for these?

Reply:
The flags are important for situations where we run the same command with different flags but the label for it is different. e.g. SFDX: Deploy Source to Org, SFDX: Deploy Source in Manifest ...